README.md

```diff
@@ -1,42 +1,24 @@
 # Funding Program Manager
 
-(* FIXME: In one or two paragraphs, describe what the extension does and why one would download it. *)
+This extension forms the basis for application management in CiviCRM: manage funding programs from application to payout.
```

Comment:
I'd suggest to use `funding application management` at this point as it is the first time the term `application` is used.